benchmarks/Symbolics/old_sparse_jacobian.jl

```diff
@@ -0,0 +1,227 @@
+function old_sparsejacobian(ops::AbstractVector, vars::AbstractVector)
+    sp = Symbolics.jacobian_sparsity(ops, vars)
```

Comment:
Should be updated to DI?

Reply:
This is a recreation of the old Symbolics AD algorithm to compare before and after. An apples-to-apples comparison would use `jacobian_sparsity` here as well, since that is what Symbolics uses.